addons/wowl/static/src/services/localization.ts

```diff
@@ -1,88 +1,149 @@
 import { DateTime } from "luxon";
-import { Service } from "../types";
-import { strftimeToLuxonFormat } from "../utils/fields";
+import { OdooEnv, Service } from "../types";
+import { parseSmartDateInput, strftimeToLuxonFormat } from "../utils/dates";
+import { escapeRegExp, intersperse, sprintf, stripAlphaDupes } from "../utils/strings";
 
-export interface Localization {
+interface LangParams {
   dateFormat: string;
   decimalPoint: string;
-  direction: string;
-  formatDateTime: (value: DateTime, format?: string, options?: { timezone: boolean }) => string;
-  grouping: any[];
-  langDateFormat: string;
-  langTimeFormat: string;
-  langDateTimeFormat: string;
+  direction: "ltr" | "rtl";
+  grouping: number[];
   multiLang: boolean;
-  parseDateTime: (value: string) => DateTime;
-  thousandsSep: string;
   timeFormat: string;
-  _t: typeof _t;
-}
-
-interface TranslatedTerms {
-  [key: string]: string;
-}
-
-/**
- * Default values for localization parameters
- */
-export function getDefaultLocalization(): Localization {
-  return {
-    dateFormat: "%m/%d/%Y",
-    decimalPoint: ".",
-    direction: "ltr",
-    formatDateTime: makeFormatDateTime(strftimeToLuxonFormat("%m/%d/%Y %H:%M:%S")),
-    grouping: [],
-    langDateFormat: strftimeToLuxonFormat("%m/%d/%Y"),
```

Comment:
make 3 constants

Reply:
in fact I already did :-)

https://github.com/odoo-dev/odoo/pull/538/files/998c7441fd92e6b12a2f1a6294edac69accab795..d1b853939b367f4b1bf5bdc368e855b38898c845#diff-4bfeeb6890f4e24db244e1da15796e0c036314900a0c194ff43e404d8eba6723R205-R209